internal/cli/kraft/kraft.go

```diff
@@ -102,6 +103,9 @@ func NewCmd() *cobra.Command {
 	cmd.AddCommand(login.NewCmd())
 	cmd.AddCommand(version.NewCmd())
 
+	cmd.AddGroup(&cobra.Group{ID: "experimental", Title: "EXPERIMENTAL COMMANDS"})
```

Comment:
Can you please remove the `cmd.AddGroup`, this should be hidden in the main menu.